packages/block-editor/src/components/rich-text/index.js

```diff
@@ -411,9 +411,9 @@ class RichTextWrapper extends Component {
 				start={ start }
 				reversed={ reversed }
 			>
-				{ ( { isSelected, value, onChange, Editable } ) =>
+				{ ( { isSelected, value, onChange, onFocus, Editable } ) =>
```

Comment:
I'm not sure about the name onFocus, it indicates that it is an event e.g: that we can pass something that is called when something gains focus, while what happens is that onFocus is a function that excitedly puts that focus back on rich text. Maybe we can call it "focus" or "focusRichText"?

Reply:
What's the difference with `onChange`? It's also not an event handlers but rather an action to change something. I'd prefer to keep it consistent with `onChange`, which is maybe also named poorly?

Reply:
Cc @aduth since I know you have opinions about this. :)